editions/tw5.com/tiddlers/images/Funding.png.meta

```diff
@@ -1,3 +1,4 @@
 title: Funding.png
 type: image/png
 tags: picture
+alt-text: Hand over a debit-card.
```

Comment:
😃

Reply:
You are right, that's not the best thing to do but IMO the image shows that. **Do you know a better description?**